lookout/style/format/analyzer.py

```diff
@@ -56,6 +56,7 @@ class FormatAnalyzer(Analyzer):
                 "remove_constant_features": True,
                 "insert_noops": False,
                 "return_sibling_indices": False,
+                "cutoff_label_support": 50,
```

Comment:
Code doesn't run with this parameter added there since __init__ of FeatureExtractor doesn't have a cutoff_label_spport and we use `**lang_params["feature_extractor"]` in several places. I suggest we have either two nested dicts in feature_extractor part of config (init and extract_features) or just another section extract_features

Reply:
Yep, discovered this after running tests